nixos/modules/services/system/nscd.nix

```diff
@@ -38,7 +38,7 @@ in
         default = if pkgs.stdenv.hostPlatform.libc == "glibc"
           then pkgs.stdenv.cc.libc.bin
           else pkgs.glibc.bin;
-        defaultText = literalExample ''
+        defaultText = lib.literalExpression ''
```

Comment:
Uses `with lib`.

```suggestion
        defaultText = literalExpression ''
```

Reply:
Sorry; does not warrant waiting for the CI again. Feel free to make a PR.